<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ADS67" name="Test" tests="13" failures="9" timestamp="14 Sep 2017 06:42:41 GMT" time="1.704" errors="0">
  <testcase name="ArthaDB_ProdCodes_Check_TC001" time="0.969" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="AUDIT-Product Codes check is Failed at UniqueID=J911620FS4904, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: AUDIT-Product Codes check is Failed at UniqueID=J911620FS4904, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_ProdCodes_Check_TC001(Arthadb_Automation.java:65)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_ProdCodes_Check_TC001 -->
  <testcase name="ArthaDB_AuditDates_Check_TC002" time="0.045" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="AUDIT-Exp and Eff Date check is Failed at UniqueID=D3527056787843525678, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: AUDIT-Exp and Eff Date check is Failed at UniqueID=D3527056787843525678, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_AuditDates_Check_TC002(Arthadb_Automation.java:109)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_AuditDates_Check_TC002 -->
  <testcase name="ArthaDB_Account_Number_Check_TC003" time="0.027" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="PAYMENTS-Account Number check is Failed at Invoice_Number=J91606706e6C122, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: PAYMENTS-Account Number check is Failed at Invoice_Number=J91606706e6C122, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Account_Number_Check_TC003(Arthadb_Automation.java:153)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Account_Number_Check_TC003 -->
  <testcase name="ArthaDB_ChargeType_Check_TC004" time="0.018" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="PAYMENTS-Charge Type check is Failed at Account_Number(s)=ACCD101N8776123, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: PAYMENTS-Charge Type check is Failed at Account_Number(s)=ACCD101N8776123, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_ChargeType_Check_TC004(Arthadb_Automation.java:197)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_ChargeType_Check_TC004 -->
  <testcase name="ArthaDB_Invoice_Number_Check_TC005" time="0.014" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="PAYMENTS-Invoice Number check is Failed at Account_Number(s)=ACCD101N9990884, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: PAYMENTS-Invoice Number check is Failed at Account_Number(s)=ACCD101N9990884, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Invoice_Number_Check_TC005(Arthadb_Automation.java:241)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Invoice_Number_Check_TC005 -->
  <testcase name="ArthaDB_Transaction_Amount_Check_TC006" time="0.016" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_ChargeSign_Check_TC007" time="0.015" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="PAYMENTS-Charge Sign Codes check is Failed at Account_Number=ACCD101N8776123, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: PAYMENTS-Charge Sign Codes check is Failed at Account_Number=ACCD101N8776123, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_ChargeSign_Check_TC007(Arthadb_Automation.java:329)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_ChargeSign_Check_TC007 -->
  <testcase name="ArthaDB_Record_Type_Check_TC008" time="0.019" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="PAYMENTS-Record Type check is Failed at Invoice_Number=J9908844423C102,J91606706e6C122, expected:&amp;lt;0&amp;gt; but was:&amp;lt;2&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: PAYMENTS-Record Type check is Failed at Invoice_Number=J9908844423C102,J91606706e6C122, expected:<0> but was:<2>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Record_Type_Check_TC008(Arthadb_Automation.java:373)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Record_Type_Check_TC008 -->
  <testcase name="ArthaDB_Customer_FirstName_Check_TC009" time="0.02" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="CUSTOMERS-First Name check is Failed at SSN=824-553-4629, expected:&amp;lt;0&amp;gt; but was:&amp;lt;1&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: CUSTOMERS-First Name check is Failed at SSN=824-553-4629, expected:<0> but was:<1>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Customer_FirstName_Check_TC009(Arthadb_Automation.java:417)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Customer_FirstName_Check_TC009 -->
  <testcase name="ArthaDB_Record_Type_Check_TC010" time="0.015" classname="exe.Arthadb_Automation"/>
  <testcase name="ArthaDB_Record_Type_Check_TC011" time="0.025" classname="exe.Arthadb_Automation">
    <failure type="junit.framework.AssertionFailedError" message="PARTNERS-Record Type check is Failed at Partner_Account_Number=15914697799,25710135206, expected:&amp;lt;0&amp;gt; but was:&amp;lt;2&amp;gt;">
      <![CDATA[junit.framework.AssertionFailedError: PARTNERS-Record Type check is Failed at Partner_Account_Number=15914697799,25710135206, expected:<0> but was:<2>
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.Assert.failNotEquals(Assert.java:329)
	at junit.framework.Assert.assertEquals(Assert.java:78)
	at junit.framework.Assert.assertEquals(Assert.java:234)
	at exe.Arthadb_Automation.ArthaDB_Record_Type_Check_TC011(Arthadb_Automation.java:508)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ArthaDB_Record_Type_Check_TC011 -->
  <testcase name="ArthaDB_Premise_City_Check_TC012" time="0.019" classname="exe.Arthadb_Automation"/>
  <testcase name="TestRunFinalProcess" time="0.417" classname="exe.ExcelRun"/>
</testsuite> <!-- Test -->
